app/uk/gov/hmrc/ngrraldfrontend/controllers/DeclarationController.scala

```diff
@@ -52,13 +53,21 @@ class DeclarationController @Inject()(declarationView: DeclarationView,
     }
   }
 
+  private def getAssessmentId(property: VMVProperty): Option[String] = {
+    property.valuations
```

Comment:
Because there will always be only one in the valuations.
```suggestion
    property.valuations.head.assessmentRef.toString
```